containers/kubernetes/how-to/revoke-kubeconfig.mdx

```diff
@@ -0,0 +1,72 @@
+---
+meta:
+  title: How to revoke a kubeconfig of a Kapsule cluster
+  description: This page explains how to revoke a kubeconfig of a Kapsule cluster
+content:
+  h1: How to revoke a kubeconfig of a Kapsule cluster
+  paragraph: This page explains how to revoke a kubeconfig of a Kapsule cluster
+tags: kubernetes kubernetes-kapsule kapsule cluster kubeconfig revoke iam
+dates:
+  validation: 2023-11-28
+  posted: 2023-11-28
+categories:
+  - kubernetes
+---
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [created](/containers/kubernetes/how-to/create-cluster) a Kubernetes Kapsule cluster
+</Message>
+
+By the end of 2023, a new Authentication (Auth) system, based on IAM tokens, is introduced. **This results in the coexistence of two authentication mechanisms.**
```

Comment:
Introduced by Scaleway ? a bit unclear (to me at least)

Reply:
Indeed by the Scaleway K8s team

Reply:
That's not something K8s native